### DISCUSSION TITLE
Create OP_DATA for output

### DISCUSSION THREAD

**Discussion by foursmalltrees** [paulmillr/micro-btc-signer#70] [Q&A]
Looking at how ordinals that is popular on bitcoin, can we use this library to embed text and even file on OP_PUSHDATA1. Looking forward if this example can be shown in this libary.

**Comment by foursmalltrees** [comment]
Thank you @paulmillr for moving this to discussion.

**Comment by paulmillr** [comment]
Ordinals got easier with 1.3.0 https://github.com/paulmillr/scure-btc-signer/releases/tag/1.3.0
